rdr_service/dao/genomics_dao.py

```diff
@@ -3452,6 +3463,86 @@ def get_event_message_informing_loop_mismatches(self, module="gem"):
 
             return records.all()
 
+    def get_event_message_results_ready_mismatches(self, module="pgx"):
+        """
+        Returns message data for records in user_event_metrics
+        but not in genomic_report_state
+        :param module: hdr, or pgx
+        :return: query result
+        """
+        module_mappings = cvl_result_reconciliation_modules
+        event_type = "result_ready"
+
+        event_names = [name for name in message_broker_report_ready_event_state_mappings.keys()
+                       if name.startswith(module)]
+
+        with self.session() as session:
+            sample_ids_subquery = self.build_set_member_sample_id_subquery("aou_wgs")
+
+            records = session.query(
+                sample_ids_subquery.c.sample_id,
+                sample_ids_subquery.c.member_id,
+                UserEventMetrics.participant_id,
+                UserEventMetrics.event_name,
+                UserEventMetrics.created_at,
+                UserEventMetrics.id.label("event_id")
+            ).outerjoin(
+                GenomicMemberReportState,
+                and_(
+                    GenomicMemberReportState.participant_id == UserEventMetrics.participant_id,
+                    GenomicMemberReportState.module == module_mappings[module],
```

Comment:
Sorry about the discrepancies in the naming conventions of this attribute on the model, this is the one normalized table without `_type` on it. Ill fix that in a later PR, just will be a large refactor/rename.